src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -170,12 +165,6 @@ const HeatmapPlot = (props) => {
     }
   }, [cellHighlight]);
 
-  useEffect(() => {
-    if (!heatmapData) return;
-
-    setVitessceData(heatmapData);
-  }, [heatmapData]);
```

Comment:
This is redundant, we can just directly use `heatmapData` in the component instead of setting it to `vitessceData`